app/controllers/concerns/strategies_concern.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+module StrategiesConcern
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :publishing?, :saving_as_draft,
+                  :render_errors, :empty_array_for,
+                  :premade_strategy
+  end
+
+  def publishing?
+    params[:publishing] == '1'
+  end
+
+  def saving_as_draft?
+    params[:publishing] != '1'
```

Comment:
You could also do `!publishing?`.

Reply:
I think we could rework the idea of `params[:publishing]` to determine the state(draft/published) of the `strategy` and rework the two methods `publishing?` and `saving_as_draft?` into a single check. 

I reckon that would be a less cumbersome refactor. I will look into that and raise an extended PR for this.

@julianguyen how does that sound?

Reply:
That's a great point! That works for me :D